aws_lambda_powertools/logging/logger.py

```diff
@@ -132,8 +132,11 @@ def __getattr__(self, name):
         # https://github.com/awslabs/aws-lambda-powertools-python/issues/97
         return getattr(self._logger, name)
 
-    def _get_log_level(self, level: str):
+    def _get_log_level(self, level: Union[str, int]) -> Union[str, int]:
         """ Returns preferred log level set by the customer in upper case """
+        if isinstance(level, int):
+            return level
+
         log_level: str = level or os.getenv("LOG_LEVEL")
```

Comment:
The docstring on line 47 above declares the LOG_LEVEL environment variable as capable of accepting an integer. That probably needs to be handled here as well.

Reply:
handling that in another PR before publishing it, as the test didn't go through